setup/install-bluespec.sh

```diff
@@ -14,24 +14,27 @@ sudo apt-get install -y ghc libghc-regex-compat-dev libghc-syb-dev \
     libghc-old-time-dev libghc-split-dev tcl-dev build-essential pkg-config \
     autoconf gperf flex bison
 
-sudo mkdir -p /opt/tools/bsc
-sudo chown $USER:$USER /opt/tools/bsc
-
 mkdir -p deps
 cd deps
 
-git clone $(python3 ${src_path}/_tools.py --tool bsc --field git-url) bsc
-cd bsc
-git checkout $(python3 ${src_path}/_tools.py --tool bsc --field git-commit)
+git clone $(python3 ${src_path}/_tools.py --tool bluespec --field git-url) bluespec
+cd bluespec
+git checkout $(python3 ${src_path}/_tools.py --tool bluespec --field git-commit)
 git submodule update --init --recursive
 
-make install-src
+make -j install-src
```

Comment:
been a while since I looked at this, will this install to $PREFIX by default if it's defined?

Reply:
Yes it does, but PREFIX is generally not defined